cmd/ndm_daemonset/probe/eventhandler.go

```diff
@@ -123,7 +124,9 @@ func (pe *ProbeEvent) deleteBlockDevice(msg controller.EventMessage) bool {
 	}
 	ok := true
 	for _, diskDetails := range msg.Devices {
-		oldBDResource := pe.Controller.GetExistingBlockDeviceResource(bdList, diskDetails.ProbeIdentifiers.Uuid)
+		UUID := strings.TrimPrefix(diskDetails.ProbeIdentifiers.Uuid, libudevwrapper.NDMDiskPrefix)
```

Comment:
can we have some function that will convert 
`disk` to `block-device`  so that we can reuse it.